### ISSUE TITLE
Android app: please reintroduce delay on first shot after pause

### ISSUE THREAD

**Issue by kaitu** [mapillary/mapillary_issues#2193]
### Basic information
* Platform: `Android`
* Version: `v 2.38` 

### Steps to reproduce behavior

 1. pause sequence by rotating the phone (pause on tilt or pause on face-down)
 2. straighten the phone to restart sequence

### Expected behavior
A post pause resume delay can be set, to allow for aiming the phone and focus the target before first photo is shot. 

### Actual behavior
A photo is shot immediately, without giving time to point the camera properly and focus the target. 

### Additional information
As I basically pause after each photo (see Issue #151 for more details), this results in at least half of the taken pictures needing to be deleted, increased memory consumption and (most annoying) longer review times.

The app used also to display a nice osd countdown, which is long gone. I can live without the countdown information. But please do reintroduce an adjustable delay on resume after pause.

**Comment by rsahlin** [comment]
Hi @kaitu 

The delay after pause affects all captured images, if a capture is skipped due to for instance tilt we want to try to take the next picture as soon as possible.

Could you please try out the distance based capture mode?
In this mode pictures will be captured by distance to previous capture (and compass angle rotation) - if you want an additional picture taken, just press the preview texture at any time.

Please let me know if this helps!

Best regards
/Richard

**Comment by kaitu** [comment]
> The delay after pause affects all captured images, if a capture is skipped due to for instance tilt we want to try to take the next picture as soon as possible.

Well, I guess most users want that. I definitely don't. What harm could a post pause delay setting cause, as long it also still allows a 0 value?

> Could you please try out the distance based capture mode?

TL;DR : no

I take all pictures manually, in that for each shot I want to decide when to take it and where to aim the camera. I know there's a manual mode exactly for this. Unfortunately, it doesn't fit my needs too well.  Fact is, I've got wabbly feet, and use to walk with the aid of walking sticks. While my hands are holding the sticks, I cannot simultaneously operate the phone in manual mode. I solved the problem using time based automatic mode operated in a sort of manually assisted form. I mounted the phone on a walking stick, in a position which offers clear view of the screen, and angled so that during normal walking the phone is facing down. Then I start a sequence and start to walk. With pause on face down active, no pictures are taken. Shouldn't be taken. Unfortunately, probably due to movements and jerks tricking the accelerometers, and fast camera reaction after a pause, I still get plenty of pictures shot at my feet. That's where the pause on tilt gets really handy, Coupling the pause on face down with the pause on tilt set to a low angle, and a reasonable post pause delay, I manage to avoid unwanted shots.
Whenever I do want to take a picture, I just raise the stick, frame the subject, and wait for the shot. My hands are always grasping the sticks, no manual intervention on the phone is needed, very fast and secure. But from the moment I start to rise the stick, I need some time to get ready for the shot. Without delay I always get at least one or two unwanted shots before a good one. Furthermore, the delay helps to avoid that unwanted short interruptions of the pause state result in an unwanted picture. BTW, it's because of this mode of operation, that I suggested #68 to try to ease the battery duration limit.

I understand my request might sound peculiar, but I didn't make it on a whim. An app setting to fit my needs would be highly appreciated.

**Comment by rsahlin** [comment]
Hi @kaitu and thanks for explaining

I do not think you made the request on a whim - I want to understand how to best solve this issue :-)
I will look into the logic behind this, please be patient.

Best regards
/Richard

**Comment by rausk** [comment]
"Could you please try out the distance based capture mode?
 In this mode pictures will be captured by distance to previous capture (and compass angle rotation) ."

The distance based capture mode seems not to work correctly, 
if you start a new sequence,, nearly always 2 or 3 pictures are taken at the same starting position.

**Comment by rsahlin** [comment]
Hi @rausk and thanks for reporting

Yes, there are usually multiple triggers when a sequence is started - this will be fixed in near future.
At the moment we are working on new UI which will be a huge improvement UX and design wise :-)

Best regards
/Richard

**Comment by Caboosey** [comment]
This is issue is also related to issue#2013.

**Comment by ovivoda** [comment]
_mapillary_issues is no longer an active repository._

_Let’s continue discussing features in our forum! We will import this issue as topic to the <a href="http://forum.mapillary.com ">Mapillary Forum</a> in the coming weeks. We’ll notify here once it’s been done. See you there!_
